tests/antares/services/api_services/test_study_api.py

```diff
@@ -38,6 +39,7 @@ class TestCreateAPI:
     study_id = "22c52f44-4c2a-407b-862b-490887f93dd8"
     antares_web_description_msg = "Mocked Server KO"
     study = Study("TestStudy", "880", ServiceFactory(api, study_id))
+    study_api = StudyApiService(api, study_id)
```

Comment:
I don't see where you use this variable you introduced. I think you can remove this line and the import that goes with it